layouts/_default/success.html

```diff
@@ -17,7 +17,7 @@ <h3 class="c-objectives__title e-heading__4">Overall Objectives</h3>
         <hr />
       {{ end }}
       <section class="c-objectives">
-        <h2 class="e-heading__4 c-objectives__title">
+        <h2 class="e-heading__4 c-objectives__title is-invisible">
```

Comment:
Why keep and hide rather than remove?

Reply:
Because there could be objectives on this page directly, so I want to indicate that some objectives are coming from another place for someone who can't necessarily tell by looking